smart_proxy_dynflow_core.gemspec

```diff
@@ -30,8 +30,8 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency('rack-test', '~> 0')
   gem.add_development_dependency('rubocop', '0.32.1')
 
-  gem.add_runtime_dependency('foreman-tasks-core', '>= 0.1')
   gem.add_runtime_dependency('dynflow', "~> 0.8.27")
+  gem.add_runtime_dependency('foreman-tasks-core', '>= 0.2.1')
```

Comment:
I believe this should be >= '0.1.5', as we backported the required changes to tasks `0.9.x` branch to keep it compatilbe with Foreman 1.15

Reply:
Right, missed that during rebase